playwright-tests/playwright.config.ts

```diff
@@ -0,0 +1,44 @@
+// *****************************************************************************
+// Copyright (C) 2021-2023 logi.cals GmbH, EclipseSource and others.
```

Comment:
change 2023 to 2024

Reply:
I did not modify this file, and I do not believe that we should assert copyright even if I did (since it's only a config file). I think it's safer not to modify the existing copyright header (i.e. not go against the original author's assertion of copyright).

Makes sense? Alternatively we could add a second copyright header, if we think we should assert copyright for eventual updates to this file.

Reply:
Since you didn't modify the file, then it's ok to keep as-is. The second header can be added later, if needed.